projects/core/src/lib/users/quota/quota-info/quota-info.component.html

```diff
@@ -0,0 +1,23 @@
+<ng-container *ngIf="hasQuota()">
+    <span class="pair">
+        <span class="label">Available credits</span>
+        <span class="value">{{ sessionQuota?.available }} of {{ sessionQuota?.total }}</span>
```

Comment:
Can there be something available of …?

Reply:
[825d5aa](https://github.com/geo-engine/geoengine-ui/pull/586/commits/825d5aa91b42a58563a534bb57f528a8d07cf0e6)

Reply:
there is no longer a total
[e46564b](https://github.com/geo-engine/geoengine-ui/pull/586/commits/e46564b203836f6ce2baf1a8d4eec605fc520333)